src/WebUI/WorkLoads/OtherWorkloadsTable.tsx

```diff
@@ -51,6 +49,7 @@ export class OtherWorkloads extends BaseComponent<IOtherWorkloadsProperties, IOt
         super(props, {});
 
         this._actionCreator = ActionsCreatorManager.GetActionCreator<WorkloadsActionsCreator>(WorkloadsActionsCreator);
```

Comment:
low prio: rename this to workloadActionCreator etc

Reply:
i will let nidhi/rajendra take the call